cmd/virt-launcher/node-labeller/node-labeller.sh

```diff
@@ -28,6 +28,8 @@ if [ -e /dev/sev ]; then
   chmod o+rw /dev/sev
 fi
 
+setcap cap_dac_override=ep /usr/libexec/qemu-kvm
```

Comment:
Why do we need this? Can we rather request this on the container level?

Reply:
@xpivarc  This is related to the reported issue: https://github.com/kubevirt/kubevirt/issues/8615,
we have tried modifying the operator.yaml, e.g.:

@@ -6954,10 +6954,9 @@ spec:
             cpu: 10m
             memory: 450Mi
         securityContext:
\-          allowPrivilegeEscalation: false
           capabilities:
\-            drop:
\-            - ALL
\+            add:
\+            - DAC_OVERRIDE
           seccompProfile:
             type: RuntimeDefault
         volumeMounts:

without effect. the 'virt-handler' won't come up, e.g.:

kubevirt      virt-handler-qvp6z                        0/1     Init:CrashLoopBackOff   6 (3m54s ago)   12m

  Warning  BackOff    22s (x48 over 10m)   kubelet            Back-off restarting failed container virt-launcher in pod virt-handler-qvp6z_kubevirt(b61fe6ba-56cf-489f-9164-bf1db9e87b5b)

do you have other suggestions on how to enable this on the container level? or do you see some issues with this change?

Reply:
@naiming-zededa The problem with this is that you are basically bypassing the permission checks, which can be dangerous. If it is a permission issue, then we should understand which one is missing